donate/templates/base_master.html

```diff
@@ -4,17 +4,19 @@
 {% get_current_language_bidi as LANGUAGE_BIDI %}
 <html lang="{{ LANGUAGE_CODE }}">
     <head>
-        <!-- Google Tag Manager -->
-        <script nonce="{{request.csp_nonce}}">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-        })(window,document,'script','dataLayer','GTM-TCB2S5V');</script>
-        <!-- End Google Tag Manager -->
         <meta charset="utf-8">
         <title>{% block title %}{% if page.seo_title %}{{ page.seo_title }}{% else %}{{ page.title }}{% endif %}{% endblock %}{% block title_suffix %} | {% trans 'Donate to Mozilla' context 'Page title' %}{% endblock %}</title>
         <meta name="description" content="{% if page.search_description %}{{ page.search_description }}{% endif %}">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        {% block datalayer_initial_script %}
```

Comment:
The reason for the GTM code being inside of a block is that this code is shared between the foundation's donate site as well as thunderbird's. And at the moment we have only been requested to track donations for the foundation site.

We are doing something similar for GA IDs in https://github.com/MozillaFoundation/donate-wagtail/pull/781